docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -1,196 +1,229 @@
+Palette supports creating and managing Kubernetes clusters deployed to an Azure account. This section guides you on how to create a Kubernetes cluster in Azure that Palette manages.
 
-Azure clusters can be created under the following scopes:
-
-* Tenant Admin
-
-* Project Scope - This is the recommended scope.
-
-Be aware that clusters that are created under the **Tenant Admin** scope are not visible under Project scope .
 
 ## Prerequisites
 
-The following prerequisites must be met before deploying a workload cluster in Azure:
-
-1. You must have an active Azure cloud account with sufficient resource limits and permissions to provision compute, network, and security resources in the desired regions.
-
-
-2. Register your Azure cloud account in Palette as described in the [Creating an Azure Cloud account](azure-cloud.md#enable-azure-cloud-account-registration-to-palette) section.
-
-
-3. A [cluster profile created](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for Azure cloud.
-
-
-<video title="azure-cluster-creation" src="/videos/clusters/public-cloud/azure/azure.mp4"></video>
+- Access to an Azure cloud account.
 
-## Deploy an Azure Cluster with Palette
+- Palette integration with Azure account. Review [Register and Manage Azure Cloud Account](azure-cloud.md) for guidance.
 
-The following steps need to be performed to provision a new Azure cluster:
+- A Secure Shell (SSH) key pre-configured in your Azure environment.
 
-1. Log in to [Palette](https://console.spectrocloud.com).
+- An infrastructure cluster profile for Azure. Review [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) for guidance.
 
+- If you want to use custom storage accounts or containers, you must create them before you create your cluster. All custom storage accounts and containers will be listed on the Cluster config page during the cluster creation process. For information about use cases for custom storage, review [Azure Storage](../azure/architecture.md/#azure-storage).
 
-2. Click on **Clusters** from the left **Main Menu**.
+  If you need help creating a custom storage account or container, check out the [Create a Storage Account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) guide or the [Manage Blob Containers](https://learn.microsoft.com/en-us/azure/storage/blobs/blob-containers-portal). 
+  
+- If you do not provide your own Virtual Network (VNet), Palette creates one for you with compute, network, and storage resources in Azure when it provisions Kubernetes clusters. Ensure there is sufficient capacity in the preferred Azure region to create the following resources. Note that Palette does not create these resources if you specify an existing VNet. 

```

Comment:
This feels a bit odd. If the user does not need to provide a VNet, then it's not a prerequisite.

Reply:
Similar wording is used/approved in the refactored AWS EKS. I think it needs to be a prereq. in the event the user doesn't provide their own VNet. I've removed the last sentence and slightly reworded to clarify:

![CleanShot 2023-11-28 at 12 00 01](https://github.com/spectrocloud/librarium/assets/117382432/c7930514-df83-4ed2-a60c-54dc0db6241e)